content/blog/noetic-eol/index.md

```diff
@@ -0,0 +1,166 @@
+
+<img src="/imgs/distro-download-data.png" alt="ROS Downloads by Distro, September 2024" style="width:600px;align:center"/>
+
+### What Does End-of-Life Mean?
+
+Users often ask us, “what does end of life mean?” 
+To put it briefly, end of life means that the ROS team will no longer support that particular ROS release. 
+In practical terms, this means that after May 31st of 2025 we will no longer be providing the following for any ROS 1 release, including Noetic:
+
+* New features or capabilities
+* Security updates
+* Bug fixes, patches or support
+* Updated binaries
+
+After May 31st, 2025 ROS 1 Noetic may find themselves exposed to un-patched security vulnerabilities and bugs, with few avenues of support short of fixing these issues themselves and recompiling Noetic from source. 
+
+It is also worth *noting the things that won’t change after ROS 1* goes end of life.
+
+* ROS 1 binaries will not suddenly disappear from our servers. They will continue to be hosted in the archive in perpetuity. 
+* ROS 1 projects will not suddenly stop working on May 31st. 
+* Users do not need to migrate immediately, but they should have migration plan and upgrade as soon as is reasonable
+
+
+<br>
+
+### ROS 2 Migration Options
+
+**If you are still using ROS 1 it is time to consider switching to ROS 2 Humble Hawksbill ([migration guide](https://docs.ros.org/en/humble/How-To-Guides/Migrating-from-ROS1.html)) or ROS 2 Jazzy Jalisco ([migration guide](https://docs.ros.org/en/jazzy/How-To-Guides/Migrating-from-ROS1.html)).**
+Which distro you choose as your migration target  is going to depend on the complexity of your ROS 1 system.
+This [migration heuristic tool](https://github.com/osrf/ros2-porting-heuristics) can provide you with estimate the complexity of  migrating your project to ROS 2.
+If have a you smaller ROS 1 project, or a reasonably de-coupled workspace, it is most likely that you can migrate ROS 2 Jazzy Jalisco directly.
+The ROS community has produced a number of migration tools which you may find helpful, including this migration tool from [AWS Labs](https://github.com/awslabs/ros2-migration-tools), and this migration tool from ROS maintainer [David Lu!](https://github.com/DLu/roscompile/tree/main/magical_ros2_conversion_tool)
+
+If your ROS 1 system is tightly coupled, has a number of un-ported dependencies, or is reasonably complex, you may want to take advantage of the [ROS 1 to ROS 2 Bridge](https://github.com/ros2/ros1_bridge).
+The ROS 1 to ROS 2 Bridge performs automated translation between major ROS versions and should allow you to migrate your project one package at a time. 
+Due to the operating system constraints for the curent active ROS releases (Noetic / Ubuntu  20.04, Humble / 22.04)  the only way to use the ROS Bridge is to use ROS 2 Humble Hawksbill compiled from source on Ubuntu 20.04, [which has limited Tier 3 support](https://www.ros.org/reps/rep-2000.html#humble-hawksbill-may-2022-may-2027).
```

Comment:
I would recommend we add a page on bridging to the ROS 1 -> ROS 2 migration guide instead of writing it here

Reply:
That could be a tough one. 

The suggestion would be to have a page, that only exists in the Humble docs branch, on how to compile Humble on 20.04 with the bridge. I could do that pretty quickly if I had some crib notes or a history dump. Would someone have something like that? Are there any gotchas?

Reply:
This is a guide I'm wanting to write, but to unblock this PR I'm good with leaving this section in for now. I will make a PR to replace it with a link when I land the bridging doc.

> The suggestion would be to have a page, that only exists in the Humble docs branch, on how to compile Humble on 20.04 with the bridge. I could do that pretty quickly if I had some crib notes or a history dump. Would someone have something like that? Are there any gotchas?

I'm thinking making it not ROS distro specific. In my mind it looks like:

1.  Pick the ROS 2 distro you want to use
2. Here's how to compile Noetic and the bridge on newer ubuntu distros
3. If something doesn't work, here's how to grab a patch from ROS O

Reply:
This would be for Humble and Jazzy in particular?

I don't have an objection but we probably want a black box warning stating that this is only recommended in the case of migration. Feel free to tag me for a review.

Reply:
Note that ROS-O can be easily installed from binaries next to all active ROS2 releases.
I suggest to replace the sentence with something along the following line:

> Due to the operating system constraints for the current active ROS releases (Noetic = Ubuntu  20.04, ROS2 >= Ubuntu 22.04) we suggest [installing ROS-One](https://ros.packages.techfak.net/) next to ROS2 on a decent Ubuntu distro (instead of building ROS2 from source on Ubuntu 20.04).